pyproject.toml

```diff
@@ -117,6 +90,7 @@ doc-no-examples = [
     "scikit-rf>=0.30.0,<1.4",
     "pyvista[io]>=0.38.0,<0.45",
```

Comment:
Not sure but can't we also get rid of matplotlib, scikit-rf and pyvista dependencies for doc build?

Reply:
@SMoraisAnsys Done